packages/engine-rn/templates/platforms/ios/Podfile

```diff
@@ -1,3 +1,7 @@
+{{PLUGIN_PODFILE_SOURCES}}
+
+{{PODFILE_HEADER}}
```

Comment:
@mihaiblaga89  prefix all injects with `INJECT_`

Reply:
switched to casting